layers/+completion/ivy/README.org

```diff
@@ -140,3 +141,37 @@ are found in [[https://oremacs.com/swiper/#minibuffer-key-bindings][the Hydra se
 | ~SPC C f~   | =counsel-colors-faces=  |
 | ~SPC C w~   | =counsel-colors-web=    |
 | ~SPC h d F~ | =counsel-describe-face= |
+
+** Search files with ivy
+=ripgrep= is recommended and =Spacemacs= will pick it up as the default seach
+app if found. To pass parameters to =ripgrep= use double dash then everything
+after it is treated as parameters for search app.
+
+For example:
+#+begin_src 
+phrase I want to search -- -tlisp
```

Comment:
I think it's -t lisp or --type lisp

Reply:
The space between the short option (one letter option, `-t`) and the argument is optional. This convention holds true for many popular linux commands.